docs/docs/resources/pipeline_config/config.yaml

```diff
@@ -0,0 +1,55 @@
+# CONFIGURATION
+#
+# The path to the folder containing the defaults file and the environment
+# defaults files.
+# REQUIRED
+defaults_path: /path/to/defaults
+# The environment you want to generate and deploy the pipeline to. Suffix your
+# environment files with this value (e.g. defaults_development.yaml for
+# environment=development).
+# REQUIRED
+environment: development
+# The kafka broker address.
+# REQUIRED
+broker: "http://localhost:9092"
+# The name of the defaults file and the prefix of the defaults environment file.
+defaults_filename_prefix: defaults
```

Comment:
I had no idea you can customize this 😄

Reply:
I'm so grateful for the schema, this would've taken soo long without it!